pkgs/top-level/all-packages.nix

```diff
@@ -5581,6 +5581,21 @@ with pkgs;
 
   gcc_multi = wrapCCMulti gcc;
 
+  llvm-multi = llvmPkgs:
+    if system == "x86_64-linux" then
+      callPackages ../development/compilers/llvm/multi.nix {
+        gcc32 = pkgsi686Linux.gcc;
+        gcc64 = pkgs.gcc;
+        llvmPackages = llvmPkgs;
+      }
+    else throw "Multilib ${cc.name} not supported on '${system}'";
+
+  gccMultiStdenv = overrideCC stdenv gcc_multi;
+  clangMultiStdenv = llvmMultiPackages.stdenv;
+  llvmMultiPackages_4 = llvm-multi llvmPackages_4;
+  llvmMultiPackages_5 = llvm-multi llvmPackages_5;
+  llvmMultiPackages = llvm-multi llvmPackages;
```

Comment:
Again, `llvmMultiPackages` does not look like a good name since it provides just `clang` and `stdenv`; and it is not clear that it should provide `clang` because it will be unused and is accessible as `stdenv.cc`.

Reply:
Good call, the name reflects my original intention while working on this... which as you very rightly point out is no longer a reasonable name for the result.

I'm opt'ing to have multi.nix return the wrapped clang-- this parallels the wrapCCMulti code and makes it possible/easy for users to install a multilib-capable clang.